stdlib/2/typing.pyi

```diff
@@ -16,6 +16,7 @@ Type = object()
 _promote = object()
 no_type_check = object()
 ClassVar = object()
+Protocol = object()
```

Comment:
I know we have a lot of similar definitions already, but I'd prefer to keep the stub closer to reality, maybe with just `class Protocol: pass`